config/install/image.style.breakpoint_2xl_2x.yml

```diff
@@ -5,10 +5,10 @@ name: breakpoint_2xl_2x
 label: 'Breakpoint - 2XL - 2x'
 effects:
   ad3f3a45-2c07-4b34-a146-e5253cadc5a1:
-    uuid: ad3f3a45-2c07-4b34-a146-e5253cadc5a1
```

Comment:
I think you still want these UUIDs. These are references to other entities.

Reply:
fixed.